### DISCUSSION TITLE
Problem with UI Rendering in the Profile Page.

### DISCUSSION THREAD

**Discussion by amannirala13** [community/community#37562] [Other Feature Feedback, Questions, & Ideas]
When I open my profile page and go to my follower's section and click on any user's profile, only the part of the UI changes (the screenshot of the issue has been provided below).

## Steps to recreate:
- Open your personal profile page
- Go to follower's section
- Click on any follower's profile.
- The UI won't update as expected.

I verified the issue on multiple browsers, on multiple systems and networks.

![image](https://user-images.githubusercontent.com/35570939/198387228-6287d3c8-f4eb-4741-b446-a91fe9540eba.png)

**Comment by radar-blip** [comment]
You're right. Same thing happening here. 
If I refresh the page it goes back to normal.

**Comment by jfullstackdev** [comment]
I cannot reproduce it, I can go exactly at the followers profile from the followers tab
